saleor/graphql/core/enums.py

```diff
@@ -92,6 +92,8 @@ def to_enum(enum_cls, *, type_name=None, **options) -> graphene.Enum:
 PermissionEnum = graphene.Enum("PermissionEnum", get_permissions_enum_list())
 TimePeriodTypeEnum = to_enum(TimePeriodType)
 ThumbnailFormatEnum = to_enum(ThumbnailFormat)
+IconThumbnailFormatEnum = to_enum(IconThumbnailFormat)
```

Comment:
Do you think that `ThumbnailFormat` and `IconThumbnailFormat` will differ in the feature? I;m wondering if we cannot use the same enum?